zone/corpse.cpp

```diff
@@ -1012,12 +1045,26 @@ bool Corpse::Process() {
 		corpse_graveyard_timer.Disable();
 		return false;
 	}
-	/*
-	if(corpse_res_timer.Check()) {
-		can_rez = false;
-		corpse_res_timer.Disable();
+	
+	//Player is offline. If rez timer is enabled, disable it and save corpse.
+	if (!is_owner_online && rezzable) {
+		if (corpse_rez_timer.Enabled()) {
+			rez_time = corpse_rez_timer.GetRemainingTime();
+			corpse_rez_timer.Disable();
+			is_corpse_changed = true;
+			Save();
+		}
+	} else if (is_owner_online && rezzable) { //Player is online. If rez timer is disabled, enable it.
+		if (corpse_rez_timer.Enabled()) {
+			rez_time = corpse_rez_timer.GetRemainingTime();
+		} else {
+			SetRezTimer();
+		}
+	}
```

Comment:
Can just wrap in a `if (rezzable)` instead of checking `rezzable` on both conditions.